addons/discuss-button/expand-width.css

```
@@ -0,0 +1,3 @@
+#navigation > .inner{
```

This doesn't work on older (`scratchr2`) pages which need different selectors.
```suggestion
#navigation > .inner,
#topnav .container {
```